lib/hexpm/web/views/layout_view.ex

```diff
@@ -42,9 +42,17 @@ defmodule Hexpm.Web.LayoutView do
       tag(:meta, property: "og:title", content: Map.get(assigns, :title)),
       tag(:meta, property: "og:type", content: "website"),
       tag(:meta, property: "og:url", content: Map.get(assigns, :canonical_url)),
-      tag(:meta, property: "og:image", content: static_url(Hexpm.Web.Endpoint, "/images/hex.png")),
+      tag(:meta, property: "og:image", content: static_url(Hexpm.Web.Endpoint, "/images/open-graph.png")),
```

Comment:
I tested with a package and they still use the big one.
<img width="503" alt="screen shot 2017-03-17 at 9 18 56 am" src="https://cloud.githubusercontent.com/assets/301291/24052658/8135577e-0af3-11e7-817d-1e791bf579af.png">